u2f-chrome-extension/messagetypes.js

```diff
@@ -18,5 +18,7 @@ var MessageTypes = {
   U2F_REGISTER_REQUEST: 'u2f_register_request',
   U2F_SIGN_REQUEST: 'u2f_sign_request',
   U2F_REGISTER_RESPONSE: 'u2f_register_response',
-  U2F_SIGN_RESPONSE: 'u2f_sign_response'
+  U2F_SIGN_RESPONSE: 'u2f_sign_response',
+  U2F_GET_API_VERSION_REQUEST: 'u2f_get_api_version_request',
```

Comment:
can you separate the addition of the getApiVersion() function into one CL and the rest of the modifications into another CL?